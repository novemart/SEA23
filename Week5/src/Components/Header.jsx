import { Link, NavLink } from 'react-router-dom'
import logo from './images/qalogo.svg'
import React from 'react'

const Header = () => {
  return (
    <header>
        <nav className='navbar navbar-expand-sm'>
            <a href="https://www.qa.com"
            className='navbar-brand' target="_blank"
            rel="noreferrer"><img src={logo} alt="QA Ltd"
            width="100"></img></a>
            <Link to="/" className='navbar-brand'>Todo App</Link>
            <div className='collapse navbar-collapse'>
              <ul className="navbar-nav mr-auto">
                <li className="navbar-item">
                  <NavLink to="/" className="nav-link" activeClassName="nav-link active">
                    All Todos
                  </NavLink>
                </li>
                <li className="navbar-item">
                  <NavLink to="/add" className="nav-link" activeClassName="nav-link active">
                    Add Todos
                  </NavLink>
                </li>
              </ul>
            </div>
        </nav>
        </header>
  )
}

export default Header