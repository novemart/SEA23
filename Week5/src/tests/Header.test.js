import React from "react";
import Header from "../Components/Header";
import { create } from "react-test-renderer";

test('Header matches snapshot', ()=>{
    const header = create(<Header />)
    expect (header.toJSON()).toMatchSnapshot();
})
