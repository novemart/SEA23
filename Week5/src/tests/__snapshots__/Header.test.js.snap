// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<header>
  <nav
    className="navbar navbar-expand-sm"
  >
    <a
      className="navbar-brand"
      href="https://www.qa.com"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="QA Ltd"
        src="qalogo.svg"
        width="100"
      />
    </a>
    <a
      className="navbar-brand"
      href="/"
    >
      Todo App
    </a>
  </nav>
</header>
`;
